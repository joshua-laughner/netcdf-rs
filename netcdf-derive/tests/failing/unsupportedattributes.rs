#[repr(C)]
#[derive(netcdf_derive::NcType)]
struct UnsupportedNetcdfAttribute1 {
    #[netcdf(gibberish)]
    A: i32,
}

#[repr(C)]
#[derive(netcdf_derive::NcType)]
#[netcdf(gibberish)]
struct UnsupportedNetcdfAttribute2 {
    A: i32,
}

#[repr(u8)]
#[derive(netcdf_derive::NcType)]
enum UnSupportedAttribute3 {
    #[netcdf(gibberish)]
    A,
}

fn main() {}
