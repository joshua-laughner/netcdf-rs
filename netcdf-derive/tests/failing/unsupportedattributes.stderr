error: NcType encountered an unknown attribute
 --> tests/failing/unsupportedattributes.rs:4:14
  |
4 |     #[netcdf(gibberish)]
  |              ^^^^^^^^^

error: NcType encountered an unknown attribute
  --> tests/failing/unsupportedattributes.rs:10:10
   |
10 | #[netcdf(gibberish)]
   |          ^^^^^^^^^

error: NcType encountered an unknown attribute
  --> tests/failing/unsupportedattributes.rs:18:14
   |
18 |     #[netcdf(gibberish)]
   |              ^^^^^^^^^
